modules/nf-core/fgbio/fastqtobam/tests/main.nf.test

```diff
@@ -0,0 +1,202 @@
+nextflow_process {
+
+    name "Test Process FGBIO_FASTQTOBAM"
+    script "../main.nf"
+    process "FGBIO_FASTQTOBAM"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "fgbio"
+    tag "fgbio/fastqtobam"
+
+    test("homo_sapiens - [fastq1, fastq2] - default") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    [
+                        file(params.test_data['homo_sapiens']['illumina']['test_umi_1_fastq_gz'], checkIfExists: true),
+                        file(params.test_data['homo_sapiens']['illumina']['test_umi_2_fastq_gz'], checkIfExists: true)
+                    ]
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.bam).match("bam")},
+                { assert snapshot(process.out.cram).match("cram")},
+                { assert snapshot(process.out.versions).match("versions") }
```

Comment:
```suggestion
                { assert snapshot(process.out).match()}
```
Why not like this?

Reply:
Good point, although the md5sums are different I noticed so I'll have some fixes coming up